sql/updates/world/3.3.5/9999_99_99_00_world.sql

```diff
@@ -0,0 +1,7 @@
+DROP TABLE IF EXISTS `playercreateinfo_cast_spell`;
+CREATE TABLE IF NOT EXISTS `playercreateinfo_cast_spell` (
```

Comment:
put the 2 files inside one plz. we don't use multiple files anymore on a single commit.